src/jdk.jlink/share/classes/jdk/tools/jmod/JmodOutputStream.java

```diff
@@ -55,16 +55,17 @@ class JmodOutputStream extends OutputStream implements AutoCloseable {
      * This method creates (or overrides, if exists) the JMOD file,
      * returning the output stream to write to the JMOD file.
      */
-    static JmodOutputStream newOutputStream(Path file, LocalDateTime date) throws IOException {
+    static JmodOutputStream newOutputStream(Path file, LocalDateTime date, int compressLevel) throws IOException {
         OutputStream out = Files.newOutputStream(file);
         BufferedOutputStream bos = new BufferedOutputStream(out);
-        return new JmodOutputStream(bos, date);
+        return new JmodOutputStream(bos, date, compressLevel);
     }
 
     private final ZipOutputStream zos;
     private final LocalDateTime date;
-    private JmodOutputStream(OutputStream out, LocalDateTime date) {
+    private JmodOutputStream(OutputStream out, LocalDateTime date, int compLevel) {
```

Comment:
Nit - I think this should be `compressLevel` just like the param name to the static method.

Reply:
Right, done so.